src/mame/jaleco/tetrisp2.cpp

```diff
@@ -983,7 +983,7 @@ static INPUT_PORTS_START( rockn )
 	PORT_BIT( 0x8000, IP_ACTIVE_LOW, IPT_UNKNOWN )
 
 	PORT_START("DSW")   //$be0008.w
-	PORT_DIPNAME( 0x0001, 0x0001, "DIPSW 1-1") // All these used to be marked 'Cheat', can't think why.
+	PORT_DIPNAME( 0x0001, 0x0001, "test") // All these used to be marked 'Cheat', can't think why.
```

Comment:
wtf the comment here

Reply:
sorry saw this https://www.reddit.com/r/MAME/comments/10kq5sx/how_to_submit_dip_switch_info/ and wanted to see how hard it would be to do a test change.

Reply:
I know you're trying to make a point, but it really is a bit of a dick move to open a joke pull request on a serious project.

Reply:
so sorry guys